docs/development/IntegrationTest.md

```diff
@@ -1,31 +1,79 @@
-<!---      
-  Licensed under the Apache License, Version 2.0 (the "License");      
-  you may not use this file except in compliance with the License.      
-  You may obtain a copy of the License at      
-      
-   http://www.apache.org/licenses/LICENSE-2.0      
-      
-  Unless required by applicable law or agreed to in writing, software      
-  distributed under the License is distributed on an "AS IS" BASIS,      
-  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.      
-  See the License for the specific language governing permissions and      
-  limitations under the License. See accompanying LICENSE file.      
+<!---
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License. See accompanying LICENSE file.
 -->
 
 # IntegrationTest
 
-## Run the existing tests.
+Submarine now supports two kind of integration tests.
+
+They are in the project's `submarine/submarine-test` directory, There are two modules, `e2e` and `test-k8s`.
+
+There are currently some differences between `e2e` and `test-k8s` in operation mode.
+
+Among them, `e2e` needs to deploy submarine locally, while `test-k8s` uses k8s to deploy submarine.
+
+These two different test methods can be applied to different test scenarios. (In the future, these two test methods may be combined or adjusted)
+
+## k8s test
+
+k8s test: When the user submits the code to his/her repository or the `apache/submarine` git repository, the travis test task will automatically start.
+
+test-k8s runs test cases in travis. It will first create a k8s cluster by using the kind tool in travis,
+
+and then compile and package the submarine project in `submarine-dist` directory to build a docker image.
+
+Then use this latest code build docker image in k8s Deploy a submarine system. Then run test case in the `test-k8s/..` directory.
```

Comment:
Then use this latest code to build a docker image and deploy a submarine system in k8s.

Reply:
Done.